### PR TITLE
Package mad.0.5.3

### PR DESCRIPTION
**Pull request by toots**
### `mad.0.5.3`
Mad decoding library
Bindings for the mad library which provides functions for encoding wave audio files into mp3

Fixes: #25704

---
* Homepage: https://github.com/savonet/ocaml-mad
* Source repo: git+https://github.com/savonet/ocaml-mad.git
* Bug tracker: https://github.com/savonet/ocaml-mad/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

File: packages/mad/mad.0.5.3/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Mad decoding library"
+description:
+  "Bindings for the mad library which provides functions for encoding wave audio files into mp3"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0"
```
Comment (samoht) [inline packages/mad/mad.0.5.3/opam:8]: Same issue with the license here

Comment (toots) [inline packages/mad/mad.0.5.3/opam:8]:
```suggestion
license: "GPL-2.0-only"
```

Comment (toots) [reply packages/mad/mad.0.5.3/opam]: Thanks. Fixed upstream.

File: packages/mad/mad.0.5.3/opam [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+synopsis: "Mad decoding library"
+description:
+  "Bindings for the mad library which provides functions for encoding wave audio files into mp3"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0-only"
+homepage: "https://github.com/savonet/ocaml-mad"
+bug-reports: "https://github.com/savonet/ocaml-mad/issues"
+depends: [
+  "conf-pkg-config" {build}
```
Comment (mseri) [inline packages/mad/mad.0.5.3/opam:12]:
```suggestion
  "ocaml" {>= "4.08"}
  "conf-pkg-config" {build}
```
due to
```
# (cd _build/default && /home/opam/.opam/4.05/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.mad.objs/byte -no-alias-deps -o src/.mad.objs/byte/mad.cmi -c -intf src/mad.mli)
# File "src/mad.mli", line 133, characters 42-52:
# Error: Unbound type constructor floatarray
```

File: packages/mad/mad.0.5.3/opam [hunk 3]
```diff
@@ -9,6 +9,7 @@ license: "GPL-2.0-only"
 homepage: "https://github.com/savonet/ocaml-mad"
 bug-reports: "https://github.com/savonet/ocaml-mad/issues"
 depends: [
+  "ocaml" {>= "4.08"}
```
Comment (toots) [inline packages/mad/mad.0.5.3/opam:12]: Applied upstream.

**Pull request merged**